src/font-size.scss

```diff
@@ -1,43 +1,90 @@
-
-$_b: 1rem; // BASE: 1rem == 16.00px
-$_r: 1.2; // RATIO
-
-$_u-1: math.pow($_r, +1) * $_b; // 1.2rem   == 19.20px
-$_u-2: math.pow($_r, +2) * $_b; // 1.44rem  == 23.04px
-$_u-3: math.pow($_r, +3) * $_b; // 1.728rem == 27.65px
-$_u-4: math.pow($_r, +4) * $_b; // 2.074rem == 33.18px
-$_u-5: math.pow($_r, +5) * $_b; // 2.488rem == 39.81px
-$_u-6: math.pow($_r, +6) * $_b; // 2.986rem == 47.78px
-$_u-7: math.pow($_r, +7) * $_b; // 3.583rem == 57.33px
-$_u-8: math.pow($_r, +8) * $_b; // 4.3rem   == 68.80px
-
-$_d-1: math.pow($_r, -1) * $_b; // 0.833rem == 13.33px
-$_d-2: math.pow($_r, -2) * $_b; // 0.694rem == 11.11px
-$_d-3: math.pow($_r, -3) * $_b; // 0.579rem == 9.26px
-
-:root, :host {
-  --font-size-base: #{$_b};
-  --font-size-ratio: #{$_r};
-  --font-size-u-8: #{$_u-8};
-  --font-size-u-7: #{$_u-7};
-  --font-size-u-6: #{$_u-6};
-  --font-size-u-5: #{$_u-5};
-  --font-size-u-4: #{$_u-4};
-  --font-size-u-3: #{$_u-3};
-  --font-size-u-2: #{$_u-2};
-  --font-size-u-1: #{$_u-1};
-  --font-size-d-1: #{$_d-1};
-  --font-size-d-2: #{$_d-2};
-  --font-size-d-3: #{$_d-3};
-  --font-size-root-clamp-ideal: 3vmin;
-  --font-size-root-clamp-min: 1em;
-  --font-size-root-clamp-max: 1.25em;
-  --font-size-root-clamp-fallback: 1em;
+@mixin font-size-5($line-height: regular) {
+  font-size: 5rem;
+  @if $line-height == regular {
+    line-height: 6.5rem;
+  }
 }
+
+@mixin font-size-4-5($line-height: regular) {
+  font-size: 4.5rem;
+  @if $line-height == regular {
+    line-height: 6rem;
+  }
+}
+
+@mixin font-size-4($line-height: regular) {
+  font-size: 4rem;
+  @if $line-height == regular {
+    line-height: 5rem;
+  }
+}
+
+@mixin font-size-3-5($line-height: regular) {
+  font-size: 3.5rem;
+  @if $line-height == regular {
+    line-height: 4.5rem;
+  }
+}
+
+@mixin font-size-3($line-height: regular) {
+  font-size: 3rem;
+  @if $line-height == regular {
+    line-height: 4rem;
+  }
+}
+
+@mixin font-size-2-5($line-height: regular) {
+  font-size: 2.5rem;
+  @if $line-height == regular {
+    line-height: 3.5rem;
+  }
+}
+
+@mixin font-size-2($line-height: regular) {
+  font-size: 2rem;
+  @if $line-height == regular {
+    line-height: 2.5rem;
+  }
+}
+
+@mixin font-size-1-75($line-height: regular) {
+  font-size: 1.75rem;
+  @if $line-height == regular {
+    line-height: 2.5rem;
+  }
+}
+
+@mixin font-size-1-5($line-height: regular) {
+  font-size: 1.5rem;
+  @if $line-height == regular {
+    line-height: 2rem;
+  }
+}
+
+@mixin font-size-1-25($line-height: regular) {
+  font-size: 1.25rem;
+  @if $line-height == regular {
+    line-height: 1.5rem;
+  }
+}
+
+@mixin font-size-1($line-height: regular) {
+  font-size: 1rem;
+  @if $line-height == regular {
+    line-height: 1.5rem;
+  }
+}
+
+@mixin font-0-75($line-height: regular) {
+  font-size: 0.75rem;
+  @if $line-height == regular {
+    line-height: 1rem;
+  }
+}
+
+@mixin font-0-5($line-height: regular) {
+  font-size: 0.5rem;
+  @if $line-height == regular {
+    line-height: 0.75rem;
+  }
+}
```

Comment:
also github is complaining that there should be an extra line at the bottom of the file